### ISSUE TITLE
npm package that includes prelude-browser

### ISSUE THREAD

**Issue by corwin-of-amber** [gkz/prelude-ls#115]

**Comment by anko** [comment]
Curious: Why do people use browser builds of JS modules?

I put stuff stuff through browserify / webpack / Rollup (to resolve `require`s), then uglify / esmangle (to minify). I need a build step for LiveScript→JS anyway, so a couple more are trivial.

Is it because you have an app where the same stable library is used across multiple dynamic pages and so you want it in a separate file with a long `Cache-Control`? You can do that with Browserify's `--standalone` and `--external` I think.

**Comment by corwin-of-amber** [comment]
Fair point. I can only answer from my personal experience (and I suspect hoping for other responses in this one random issue thread is highly optimistic); I don't use browserify / webpack / etc. and perhaps that's my own fallacy. I usually have just a handful of `.ls` files so I just compile them with `lsc` and include them in the page.

I believe this simple model should still be possible, at least for the sake of those at entry level who are not familiar with bundlers and want to experiment with LiveScript.
